### DISCUSSION TITLE
Any issues for beginners?

### DISCUSSION THREAD

**Discussion by brunobeeee** [holger1411/understrap#1776] [Q&A]
Hello,

We are a group of three students from Berlin. In the next few weeks we are part of a project at our university where we want to contribute to an open source project. So we decided on understrap. We are happy to be here! :)
We are new to open source development and wordpress themes so any recommendations of beginner friendly issues we could solve would be appreciated!

Thanks in advance :)

**Comment by bacoords** [comment]
Welcome! There are a few ways to get started - 
- Read the [instructions on contributing](https://github.com/understrap/understrap/blob/main/.github/CONTRIBUTING.md) to this project
- Review issues with the [Help Wanted](https://github.com/understrap/understrap/labels/Help%20wanted%21) label and offer any support
- Review issues with the [Good First Issue](https://github.com/understrap/understrap/labels/Good%20First%20Issue) label and submit a PR
- Browse the [discussion board](https://github.com/understrap/understrap/discussions) and offer any responses
- Review the [documentation](https://github.com/understrap/understrap.github.io) and flag any issues or submit any PRs for improvement.
